tests/listobjects/matrix_ttus.go

```diff
@@ -53,4 +53,48 @@ var ttus = []matrixTest{
 			},
 		},
 	},
+	{
+		Name: "ttus_recursive",
+		Tuples: []*openfgav1.TupleKey{
+			{Object: "ttus:recursive_1", Relation: "ttu_recursive", User: "user:recursive_anne"},
+
+			{Object: "ttus:recursive_2", Relation: "ttu_parent", User: "ttus:recursive_1"},
+			{Object: "ttus:recursive_3", Relation: "ttu_parent", User: "ttus:recursive_2"},
+
```

Comment:
It will also be interesting to have test where the same object can be accessible because it appears in different path

For example
```
{Object: "ttus:recursive_1_1", Relation: "ttu_recursive", User: "user:recursive_bob"},
{Object: "ttus:recursive_2_1", Relation: "ttu_parent", User: "ttus:recursive_1_1"},
{Object: "ttus:recursive_3_1", Relation: "ttu_parent", User: "ttus:recursive_2_1"},
{Object: "ttus:recursive_3_1", Relation: "ttu_recursive", User: "user:recursive_bob"},
```
We should ensure that ListObjects for `user:recursive_bob` will see `ttus:recursive_1_1`, `ttus:recursive_2_1`, `ttus:recursive_3_1`.  Notice that `ttus:recursive_3_1` did not appear twice.